seedDB/README.md

```diff
@@ -1,25 +1,7 @@
-Files to seed a new cBioPortal database.
+These files are MySQL database dump for seeding a new instance of the cBioPortal DB. They contain all the necessary background data for a properly functioning cBioPortal website, including cancer types, gene, uniprot-mappings, drug, pdb and network data. Data is compatible with release **1.3.1** of cBioPortal.  
```

Comment:
Good point. I updated the text to make the versioning story more explicit. 
Regarding cgds.sql at 1.3.0, this is not a mistake. The migration.sql is also at this same 1.3.0: see https://github.com/cBioPortal/cbioportal/blob/v1.3.1/core/src/main/resources/db/migration.sql